examples/mockups/product.html

```diff
@@ -59,43 +59,29 @@ <h1 class="ui header massive"><<i>missing</i>> Infographic</h1>
               <h3 class="ui header inverted"><<i>missing</i>> Illustration</h3>
               <h3 class="ui header inverted">Docs as Code</h3>
               <p>
-                Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx and Mkdocs, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with Pull Request previews.
+                Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx, Mkdocs, and Jupyter Book, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with Pull Request previews.
```

Comment:
Pull request is specific to GitHub, but they do not reference it as a proper noun:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

```suggestion
                Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx, Mkdocs, and Jupyter Book, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with pull request previews.
```